{"uid":"15826cb28ed1f808","name":"test_pre_upgrade_check","fullName":"tests.integration.ha_tests.test_upgrade_from_stable#test_pre_upgrade_check","historyId":"6f73cd86130887313b53a62d2e64791e","time":{"start":1734227104958,"stop":1734227104958,"duration":0},"description":"Test that the pre-upgrade-check action runs successfully.","descriptionHtml":"<p>Test that the pre-upgrade-check action runs successfully.</p>\n","status":"skipped","statusMessage":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_setup\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 840, in pytest_runtest_setup\n    yield from self._runtest_for(item, \"setup\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 875, in pytest_runtest_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_setup\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 160, in pytest_runtest_setup\n    item.session._setupstate.setup(item)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 514, in setup\n    col.setup()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1630, in setup\n    self._request._fillfixtures()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 697, in _fillfixtures\n    item.funcargs[argname] = self.getfixturevalue(argname)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 532, in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 617, in _get_active_fixturedef\n    fixturedef.execute(request=subrequest)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1091, in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n_pytest.outcomes.XFailed: aborted","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1734226501364,"stop":1734226501365,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1734226501365,"stop":1734226501365,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1734226501365,"stop":1734226501599,"duration":234},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1734227104958,"stop":1734227104958,"duration":0},"status":"failed","statusMessage":"_pytest.outcomes.XFailed: aborted\n","statusTrace":"  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_operator/plugin.py\", line 234, in abort_on_fail\n    pytest.xfail(\"aborted\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-k8s-operator/test-runners-2-github-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/outcomes.py\", line 206, in xfail\n    raise XFailed(reason)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"ops_test::finalizer","time":{"start":1734227104962,"stop":1734227105288,"duration":326},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1734227105288,"stop":1734227105289,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1734227105289,"stop":1734227105289,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1734227105289,"stop":1734227105289,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"@pytest.mark.skipif(False, reason='Requires amd64 architecture')"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_upgrade_from_stable"},{"name":"host","value":"fv-az1530-5"},{"name":"thread","value":"13180-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_upgrade_from_stable"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"fee7904667c88fb","status":"unknown","time":{}}],"categories":[],"history":{"statistic":{"failed":0,"broken":0,"skipped":5,"passed":181,"unknown":3,"total":189},"items":[{"uid":"a91d933be45c169","reportUrl":"../267//#testresult/a91d933be45c169","status":"passed","time":{"start":1734140264677,"stop":1734140281866,"duration":17189}},{"uid":"a160b1a29afc2aaa","reportUrl":"../266//#testresult/a160b1a29afc2aaa","status":"passed","time":{"start":1734054269025,"stop":1734054289249,"duration":20224}},{"uid":"4e64cb421d2f142a","reportUrl":"../265//#testresult/4e64cb421d2f142a","status":"passed","time":{"start":1733967594479,"stop":1733967611657,"duration":17178}},{"uid":"6344b15feea16edd","reportUrl":"../264//#testresult/6344b15feea16edd","status":"passed","time":{"start":1733881208275,"stop":1733881225464,"duration":17189}},{"uid":"d8e7287e885a2bf2","reportUrl":"../263//#testresult/d8e7287e885a2bf2","status":"passed","time":{"start":1733794882831,"stop":1733794903076,"duration":20245}},{"uid":"27abc2c3ca3a3791","reportUrl":"../262//#testresult/27abc2c3ca3a3791","status":"passed","time":{"start":1733708189262,"stop":1733708206478,"duration":17216}},{"uid":"a276fecf2f14efd8","reportUrl":"../261//#testresult/a276fecf2f14efd8","status":"passed","time":{"start":1733621819957,"stop":1733621840190,"duration":20233}},{"uid":"39e45d4c9c7c02a6","reportUrl":"../260//#testresult/39e45d4c9c7c02a6","status":"passed","time":{"start":1733535385142,"stop":1733535404389,"duration":19247}},{"uid":"4c197db52482742e","reportUrl":"../259//#testresult/4c197db52482742e","status":"passed","time":{"start":1733448960567,"stop":1733448980802,"duration":20235}},{"uid":"2d443b49275408b0","reportUrl":"../258//#testresult/2d443b49275408b0","status":"passed","time":{"start":1733362685014,"stop":1733362704276,"duration":19262}},{"uid":"c45cdd57a6071b5a","reportUrl":"../257//#testresult/c45cdd57a6071b5a","status":"passed","time":{"start":1733276173120,"stop":1733276193347,"duration":20227}},{"uid":"6c1bfc518232b810","reportUrl":"../256//#testresult/6c1bfc518232b810","status":"passed","time":{"start":1733190256341,"stop":1733190273520,"duration":17179}},{"uid":"a28c2693adcd6f98","reportUrl":"../255//#testresult/a28c2693adcd6f98","status":"passed","time":{"start":1733103316426,"stop":1733103333604,"duration":17178}},{"uid":"8d20429bcc0685ce","reportUrl":"../254//#testresult/8d20429bcc0685ce","status":"passed","time":{"start":1733017355516,"stop":1733017372694,"duration":17178}},{"uid":"c776982693434cf8","reportUrl":"../253//#testresult/c776982693434cf8","status":"passed","time":{"start":1732930907981,"stop":1732930929286,"duration":21305}},{"uid":"dd4c569524814ef","reportUrl":"../252//#testresult/dd4c569524814ef","status":"passed","time":{"start":1732844303172,"stop":1732844323414,"duration":20242}},{"uid":"28fbb7a236f772ee","reportUrl":"../251//#testresult/28fbb7a236f772ee","status":"passed","time":{"start":1732758324433,"stop":1732758345746,"duration":21313}},{"uid":"a142969f3b7600ac","reportUrl":"../250//#testresult/a142969f3b7600ac","status":"passed","time":{"start":1732671979274,"stop":1732671996478,"duration":17204}},{"uid":"49cd4bb66efe06d","reportUrl":"../249//#testresult/49cd4bb66efe06d","status":"passed","time":{"start":1732585303874,"stop":1732585325176,"duration":21302}},{"uid":"a9c9045726f6e958","reportUrl":"../248//#testresult/a9c9045726f6e958","status":"passed","time":{"start":1732498671094,"stop":1732498689300,"duration":18206}}]},"tags":["asyncio","abort_on_fail","@pytest.mark.skipif(False, reason='Requires amd64 architecture')","group(1)"]},"source":"15826cb28ed1f808.json","parameterValues":[]}